rskj-core/src/test/java/co/rsk/net/SnapshotProcessorTest.java

```diff
@@ -333,6 +334,35 @@ void processStateChunkRequestInternal(Peer sender, SnapStateChunkRequestMessage
         assertEquals(msg, jobArg.getValue().getMsg());
     }
 
+    @Test
```

Comment:
suggestion: 

Is it possible to replicate this error without so many coupled mocks? Isn't possible to execute the test like the others one and just make the `peersInformation` throws an exception to trigger the method you want? Something like:

`doThrow(new RuntimeException("Some error")).when(peersInformation.getOrRegisterPeer(any()));`

Reply:
We can improve that in a future ticket. I am closing the conversation now.